perfmetrics/scripts/hns_rename_folders_metrics/generate_folders_and_files.py

```diff
@@ -95,7 +95,140 @@ def list_directory(path) -> list:
     return contents_url
   except subprocess.CalledProcessError as e:
     logmessage(e.output.decode('utf-8'))
-    subprocess.call('bash', shell=True)
+
+
+def compare_folder_structure(folder, folder_url) -> bool:
```

Comment:
Python follows the naming convention of "_" for private functions: https://google.github.io/styleguide/pyguide.html#3162-naming-conventions

Do you want to follow that?

Reply:
Changed to follow this convention.